.github/workflows/unit-tests.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.15.x, 1.16.x]
+        go-version: [1.15.x, 1.18.x]
```

Comment:
I think we require go 1.17 as the minimum, so would it not make more sense to test with 1.17 and 1.18 here?

Reply:
I tried making it work similar to https://github.com/googleapis/google-cloud-go/blob/main/spanner/go.mod#L3, In client library too we do test with earliest(1.15) and latest(1.18) Go versions on each presubmit.

Reply:
But why is 1.15 earliest in this case? Aren't the other updates in this PR setting the minimum required version of Go to 1.17?

Reply:
I guess because we want to follow this https://cloud.google.com/go/getting-started/supported-go-versions